test/ShengJi.js

```diff
@@ -170,41 +170,88 @@ describe('ShengJi', function(suite) {
-      '4 of diamonds is stronger than 3 of diamonds');
+      t.true(ShengJi.isStronger(threeDiamonds, fourDiamonds, trumpLevel, trumpSuit, leadingSuit),
+        '4 of diamonds is stronger than 3 of diamonds');
 
-    t.false(ShengJi.isStronger(threeDiamonds, twoClubs, trumpLevel, trumpSuit, leadingSuit),
-      '2 of clubs (not trump) is not stronger than 3 of diamonds');
+      t.false(ShengJi.isStronger(threeDiamonds, twoClubs, trumpLevel, trumpSuit, leadingSuit),
+        '2 of clubs (not trump) is not stronger than 3 of diamonds');
 
-    t.false(ShengJi.isStronger(threeDiamonds, threeClubs, trumpLevel, trumpSuit, leadingSuit),
-      '3 of clubs (not trump) is not stronger than 3 of diamonds');
+      t.false(ShengJi.isStronger(threeDiamonds, threeClubs, trumpLevel, trumpSuit, leadingSuit),
+        '3 of clubs (not trump) is not stronger than 3 of diamonds');
 
-    t.true(ShengJi.isStronger(threeDiamonds, twoTrump, trumpLevel, trumpSuit, leadingSuit),
-      '2 of spades (trump) is stronger than 3 of diamonds');
+      t.true(ShengJi.isStronger(threeDiamonds, twoTrump, trumpLevel, trumpSuit, leadingSuit),
+        '2 of spades (trump) is stronger than 3 of diamonds');
 
-    t.true(ShengJi.isStronger(threeDiamonds, threeTrump, trumpLevel, trumpSuit, leadingSuit),
-      '3 of spades (trump) is stronger than 3 of diamonds');
+      t.true(ShengJi.isStronger(threeDiamonds, threeTrump, trumpLevel, trumpSuit, leadingSuit),
+        '3 of spades (trump) is stronger than 3 of diamonds');
 
-    t.false(ShengJi.isStronger(threeTrump, threeDiamonds, trumpLevel, trumpSuit, leadingSuit),
-      '3 of diamonds is not stronger than 3 of spades (trump)');
+      t.false(ShengJi.isStronger(threeTrump, threeDiamonds, trumpLevel, trumpSuit, leadingSuit),
+        '3 of diamonds is not stronger than 3 of spades (trump)');
 
-    t.true(ShengJi.isStronger(threeTrump, currentLevel, trumpLevel, trumpSuit, leadingSuit),
-      'Current level is stronger than 3 of spades (trump)');
+      t.true(ShengJi.isStronger(threeTrump, currentLevel, trumpLevel, trumpSuit, leadingSuit),
+        'Current level is stronger than 3 of spades (trump)');
 
-    t.true(ShengJi.isStronger(currentLevel, currentLevelAndSuit, trumpLevel, trumpSuit, leadingSuit),
-      'Current level and suit is stronger than just current level');
+      t.true(ShengJi.isStronger(currentLevel, currentLevelAndSuit, trumpLevel, trumpSuit, leadingSuit),
+        'Current level and suit is stronger than just current level');
 
-    t.true(ShengJi.isStronger(currentLevelAndSuit, blackJoker, trumpLevel, trumpSuit, leadingSuit),
-      'Black joker is stronger than current level and suit');
+      t.true(ShengJi.isStronger(currentLevelAndSuit, blackJoker, trumpLevel, trumpSuit, leadingSuit),
+        'Black joker is stronger than current level and suit');
+
+      t.true(ShengJi.isStronger(blackJoker, redJoker, trumpLevel, trumpSuit, leadingSuit),
+        'Red joker is stronger than black joker');
```

Comment:
If you want to use `> or <` instead ` is stronger than ` you can replace these too.
Yeah should standardize that if we want.